frontend/public/components/sidebars/resource-sidebar.jsx

```diff
@@ -22,35 +14,19 @@ const sidebarScrollTop = () => {
 class ResourceSidebarWrapper extends React.Component {
   constructor(props) {
     super(props);
-    this.toggleSidebar = this.toggleSidebar.bind(this);
-    this.state = {
-      showSidebar: !props.startHidden,
-    };
+    this.handleClick = this.handleClick.bind(this);
   }
 
-  toggleSidebar() {
-    this.setState(
-      (state) => {
-        return { showSidebar: !state.showSidebar };
-      },
-      () => window.dispatchEvent(new Event('sidebar_toggle')),
-    );
+  handleClick() {
```

Comment:
This needs a more specific name. But why do we need this method at all? Can't we just pass `this.props.toggleSidebar` directly below?